.github/workflows/build_windows_packages.yml

```diff
@@ -70,7 +70,7 @@ jobs:
             -DTHEROCK_ENABLE_COMM_LIBS=OFF \
             -DTHEROCK_ENABLE_MATH_LIBS=OFF \
             -DTHEROCK_ENABLE_ML_LIBS=OFF \
-            -DTHEROCK_ENABLE_HIPIFY=OFF \
+            -DTHEROCK_ENABLE_HIPIFY=ON \
```

Comment:
Note to self: merge conflicts between this and https://github.com/nod-ai/TheRock/pull/103

* keep the table updated
* sort the options in this workflow by component